app/models/patient.rb

```diff
@@ -122,15 +122,50 @@ def latest_transfer
 
   # Patients who are eligible for reminders
   scope :reminder_eligible, lambda {
-    where(purged: false)
-      .where(pause_notifications: false)
+    monitoring_open
+      .joins(:dependents)
       .where('patients.id = patients.responder_id')
-      .where.not('latest_assessment_at >= ?', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
+      .where.not(preferred_contact_method: ['Unknown', 'Opt-out', '', nil])
+      .where(pause_notifications: false)
+      .where(
+        'patients.isolation = ? '\
+        'OR patients.continuous_exposure = ? '\
+        'OR patients.last_date_of_exposure >= ? '\
+        'OR (patients.last_date_of_exposure IS NULL AND patients.created_at >= ?)',
+        true,
+        true,
+        ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day,
+        ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day
+      )
+      .where(
+        'patients.latest_assessment_at < ? '\
+        'OR patients.latest_assessment_at IS NULL',
+        Time.now.getlocal('-04:00').beginning_of_day
+      )
+      .where(
+        'patients.last_assessment_reminder_sent <= ? '\
+        'OR patients.last_assessment_reminder_sent IS NULL',
+        12.hours.ago
+      )
       .or(
-        where(purged: false)
-          .where(pause_notifications: false)
+        joins(:dependents)
+          .where(purged: false)
+          .where(head_of_household: true)
           .where('patients.id = patients.responder_id')
-          .where(latest_assessment_at: nil)
+          .where('dependents_patients.id != dependents_patients.responder_id')
+          .where.not(preferred_contact_method: ['Unknown', 'Opt-out', '', nil])
+          .where(pause_notifications: false)
+          .where('dependents_patients.monitoring = ?', true)
+          .where(
+            'dependents_patients.isolation = ? '\
+            'OR dependents_patients.continuous_exposure = ? '\
+            'OR dependents_patients.last_date_of_exposure >= ? '\
+            'OR (dependents_patients.last_date_of_exposure IS NULL AND dependents_patients.created_at >= ?)',
+            true,
+            true,
+            ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day,
+            ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day
+          )
```

Comment:
Not checking if a dependent has already been sent a reminder today or if a report has already been submitted for a dependent. This may want to be done in the future - and I don't think will be too difficult to add to the scope.

Reply:
Dependents do not have last reminder sent set on them, only HoH.

Checking if a report has already been submitted for a dependent is valid. The logic is duplicated from the HoH to the dependent:

```
     .where(
        'dependents_patients.latest_assessment_at < ? '\
        'OR dependents_patients.latest_assessment_at IS NULL',
        Time.now.getlocal('-04:00').beginning_of_day
```

Reply:
I can add this in, but before I do I just want to verify that this would be intended functionality of dependents. @holmesie I think you said to hold on something like this but maybe this is relevant now.

Reply:
This is a good question and something we need to discuss with the PO team before making the change, as it has implications (like preventing a dependent from getting a reminder - through the HoH - if a user has manually reported for them). This makes sense to me and I think would be a more consistent change, we just have to verify with the team. Let's discuss internally.